From e478371e0875558f34e43e863bc1b352b6d03311 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Wed, 7 Jul 2021 21:17:08 -0700
Subject: [PATCH 195/230] styles: Use user fonts for Material UI themes

Only using user-selected fonts for a subset of system apps leads to an
inconsistent visual experience, so apply them to the system's Material
UI themes for consistency.

Change-Id: Ib0be558530ae101fae2a5c9e05610659a87cbae9
---
 .../res/values/donottranslate_material.xml    | 22 +++++++++----------
 core/res/res/values/strings.xml               | 10 ++++-----
 core/res/res/values/styles.xml                |  4 ++--
 core/res/res/values/styles_material.xml       | 14 ++++++------
 packages/PrintSpooler/res/values/styles.xml   |  2 +-
 5 files changed, 26 insertions(+), 26 deletions(-)

diff --git a/core/res/res/values/donottranslate_material.xml b/core/res/res/values/donottranslate_material.xml
index 9cf9f6cfa86..aa6693ad750 100644
--- a/core/res/res/values/donottranslate_material.xml
+++ b/core/res/res/values/donottranslate_material.xml
@@ -17,16 +17,16 @@
 <resources>
 
     <string name="font_family_display_4_material">sans-serif-light</string>
-    <string name="font_family_display_3_material">sans-serif</string>
-    <string name="font_family_display_2_material">sans-serif</string>
-    <string name="font_family_display_1_material">sans-serif</string>
-    <string name="font_family_headline_material">sans-serif</string>
-    <string name="font_family_title_material">sans-serif-medium</string>
-    <string name="font_family_subhead_material">sans-serif</string>
-    <string name="font_family_menu_material">sans-serif</string>
-    <string name="font_family_body_2_material">sans-serif-medium</string>
-    <string name="font_family_body_1_material">sans-serif</string>
-    <string name="font_family_caption_material">sans-serif</string>
-    <string name="font_family_button_material">sans-serif-medium</string>
+    <string name="font_family_display_3_material">@string/config_bodyFontFamily</string>
+    <string name="font_family_display_2_material">@string/config_bodyFontFamily</string>
+    <string name="font_family_display_1_material">@string/config_bodyFontFamily</string>
+    <string name="font_family_headline_material">@string/config_bodyFontFamily</string>
+    <string name="font_family_title_material">@string/config_bodyFontFamilyMedium</string>
+    <string name="font_family_subhead_material">@string/config_bodyFontFamily</string>
+    <string name="font_family_menu_material">@string/config_bodyFontFamily</string>
+    <string name="font_family_body_2_material">@string/config_bodyFontFamilyMedium</string>
+    <string name="font_family_body_1_material">@string/config_bodyFontFamily</string>
+    <string name="font_family_caption_material">@string/config_bodyFontFamily</string>
+    <string name="font_family_button_material">@string/config_bodyFontFamilyMedium</string>
 
 </resources>
diff --git a/core/res/res/values/strings.xml b/core/res/res/values/strings.xml
index 5c659123b02..0c28a2a03ef 100644
--- a/core/res/res/values/strings.xml
+++ b/core/res/res/values/strings.xml
@@ -4808,16 +4808,16 @@
     <string name="time_placeholder">--</string>
 
     <!-- DO NOT TRANSLATE -->
-    <string name="radial_numbers_typeface">sans-serif</string>
+    <string name="radial_numbers_typeface">@string/config_bodyFontFamily</string>
     <!-- DO NOT TRANSLATE -->
-    <string name="sans_serif">sans-serif</string>
+    <string name="sans_serif">@string/config_bodyFontFamily</string>
 
     <!-- DO NOT TRANSLATE -->
-    <string name="date_picker_month_typeface">sans-serif-medium</string>
+    <string name="date_picker_month_typeface">@string/config_bodyFontFamilyMedium</string>
     <!-- DO NOT TRANSLATE -->
-    <string name="date_picker_day_of_week_typeface">sans-serif-medium</string>
+    <string name="date_picker_day_of_week_typeface">@string/config_bodyFontFamilyMedium</string>
     <!-- DO NOT TRANSLATE -->
-    <string name="date_picker_day_typeface">sans-serif-medium</string>
+    <string name="date_picker_day_typeface">@string/config_bodyFontFamilyMedium</string>
 
     <!-- Lock-to-app unlock pin string -->
     <string name="lock_to_app_unlock_pin">Ask for PIN before unpinning</string>
diff --git a/core/res/res/values/styles.xml b/core/res/res/values/styles.xml
index f920083f5cb..bfbead6d222 100644
--- a/core/res/res/values/styles.xml
+++ b/core/res/res/values/styles.xml
@@ -960,12 +960,12 @@ please see styles_device_defaults.xml.
     </style>
 
     <style name="TextAppearance.Toast">
-        <item name="fontFamily">sans-serif</item>
+        <item name="fontFamily">@string/config_bodyFontFamily</item>
         <item name="textSize">14sp</item>
     </style>
 
     <style name="TextAppearance.Tooltip">
-        <item name="fontFamily">sans-serif</item>
+        <item name="fontFamily">@string/config_bodyFontFamily</item>
         <item name="textSize">14sp</item>
     </style>
 
diff --git a/core/res/res/values/styles_material.xml b/core/res/res/values/styles_material.xml
index f380950366a..7b3617b7a95 100644
--- a/core/res/res/values/styles_material.xml
+++ b/core/res/res/values/styles_material.xml
@@ -429,7 +429,7 @@ please see styles_device_defaults.xml.
     <style name="TextAppearance.Material.TimePicker.InputHeader" parent="TextAppearance.Material">
         <item name="textSize">@dimen/text_size_display_1_material</item>
         <item name="textColor">@color/white</item>
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">@string/config_bodyFontFamilyMedium</item>
     </style>
 
     <style name="TextAppearance.Material.TimePicker.InputField" parent="TextAppearance.Material">
@@ -438,31 +438,31 @@ please see styles_device_defaults.xml.
 
     <style name="TextAppearance.Material.TimePicker.PromptLabel" parent="TextAppearance.Material">
         <item name="textSize">@dimen/timepicker_text_size_normal</item>
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">@string/config_bodyFontFamilyMedium</item>
     </style>
 
     <style name="TextAppearance.Material.DatePicker.YearLabel" parent="TextAppearance.Material">
         <item name="textColor">@color/primary_text_secondary_when_activated_material_inverse</item>
         <item name="textSize">@dimen/date_picker_year_label_size</item>
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">@string/config_bodyFontFamilyMedium</item>
     </style>
 
     <style name="TextAppearance.Material.DatePicker.DateLabel" parent="TextAppearance.Material">
         <item name="textColor">@color/primary_text_secondary_when_activated_material_inverse</item>
         <item name="textSize">@dimen/date_picker_date_label_size</item>
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">@string/config_bodyFontFamilyMedium</item>
     </style>
 
     <style name="TextAppearance.Material.DatePicker.List.YearLabel" parent="TextAppearance.Material">
         <item name="textColor">?attr/textColorPrimary</item>
         <item name="textSize">@dimen/datepicker_list_year_label_size</item>
-        <item name="fontFamily">sans-serif</item>
+        <item name="fontFamily">@string/config_bodyFontFamily</item>
     </style>
 
     <style name="TextAppearance.Material.DatePicker.List.YearLabel.Activated">
         <item name="textColor">?attr/colorControlActivated</item>
         <item name="textSize">@dimen/datepicker_list_year_activated_label_size</item>
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">@string/config_bodyFontFamilyMedium</item>
     </style>
 
     <style name="TextAppearance.Material.Notification">
@@ -473,7 +473,7 @@ please see styles_device_defaults.xml.
     <style name="TextAppearance.Material.Notification.Reply" />
 
     <style name="TextAppearance.Material.Notification.Title">
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">@string/config_bodyFontFamilyMedium</item>
         <item name="textSize">@dimen/notification_title_text_size</item>
     </style>
 
diff --git a/packages/PrintSpooler/res/values/styles.xml b/packages/PrintSpooler/res/values/styles.xml
index 1e63a67e50a..5d8e244e5a9 100644
--- a/packages/PrintSpooler/res/values/styles.xml
+++ b/packages/PrintSpooler/res/values/styles.xml
@@ -28,7 +28,7 @@
         <item name="android:layout_marginTop">16dip</item>
         <item name="android:layout_marginBottom">16dip</item>
         <item name="android:textColor">?android:attr/colorAccent</item>
-        <item name="android:fontFamily">sans-serif-medium</item>
+        <item name="android:fontFamily">@*android:string/config_bodyFontFamilyMedium</item>
         <item name="android:textSize">14sp</item>
     </style>
 </resources>
-- 
2.25.1

